BOOTNODE.md

```diff
@@ -1,5 +1,14 @@
 # Description
 
+A "Bootnode" is a match-maker for Whisper nodes like a [Mailserver](/MAILSERVER.md). It helps a new node find other nodes it can communicate with. It bootstraps connections between them.
```

Comment:
```suggestion
A "Bootnode" is a match-maker for any type of nodes. It helps a new node find other nodes it can communicate with. It bootstraps connections between them.
```

Reply:
Fixed.